### PR TITLE
docs(severity): Add docs for high priority alert condition

### PR DESCRIPTION
**Pull request by snigdhas**
DO NOT MERGE

This is a temporary docs page for the high priority alerts condition, intended for LA users

[Preview](https://sentry-docs-git-snigdha-priority-docs.sentry.dev/product/alerts/create-alerts/high-priority-alerts/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-docs%2Fpull%2F8845)

### CODE REVIEW

File: src/docs/product/alerts/create-alerts/high-priority-alerts.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Priority-based Alerts in Sentry
+
+Sentry is rolling out a set of updates to make alerts more actionable. Rather than defaulting to sending alerts on every new issue, we’ll only notify on issues that pass a priority threshold based on historical actions on similar issues.
+
+To start, we’ll modify the default alerts with Sentry projects with a new condition that filters alerts to only high-priority issues. Each new issue will have a score calculated for the importance of the first event. If that first event is deemed to be high severity or has a fatal log level, we'll alert you. If the first event is low severity, we'll only alert you if the issue is forecasted to worsen over time.
+
+In summary, the high priority alert condition will notify you if an issue is:
+
+- Similar to issues that have been viewed & responded to quickly in the past, or likely to be actionable (read on for more details)
+- Escalating in volume: for new issues, we’ll compare the hourly event rate to the 95th percentile hourly event rate for new issues in the project
```
Comment (isabellaenriquez) [inline src/docs/product/alerts/create-alerts/high-priority-alerts.mdx:10]: not sure how specific we want to be with the semantics but the 95th percentile hourly event rate can include non-new issues; the condition to be included is to have more than 1 event in the past week and the issue itself being older than an hour

File: src/docs/product/alerts/create-alerts/high-priority-alerts.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Priority-based Alerts in Sentry
+
+Sentry is rolling out a set of updates to make alerts more actionable. Rather than defaulting to sending alerts on every new issue, we’ll only notify on issues that pass a priority threshold based on historical actions on similar issues.
+
+To start, we’ll modify the default alerts with Sentry projects with a new condition that filters alerts to only high-priority issues. Each new issue will have a score calculated for the importance of the first event. If that first event is deemed to be high severity or has a fatal log level, we'll alert you. If the first event is low severity, we'll only alert you if the issue is forecasted to worsen over time.
+
```
Comment (lizokm) [inline src/docs/product/alerts/create-alerts/high-priority-alerts.mdx:6]:
```suggestion
The current default alerts within Sentry projects will be modified with a new condition that will filter alerts to only notify on high-priority issues. We'll calculate an importance score for the first event of each new issue. If that first event is deemed to be high-severity or has a fatal log level, we'll alert you. If the first event is low-severity, we'll only alert you if the issue is forecasted to get worse over time.

```

File: src/docs/product/alerts/create-alerts/high-priority-alerts.mdx (hunk 1 above)
Comment (lizokm) [inline src/docs/product/alerts/create-alerts/high-priority-alerts.mdx:10]:
```suggestion
The high-priority alert condition will notify you if an issue is:

- Similar to issues that have been viewed and responded to quickly in the past, or are likely to be actionable. (Read on for more details.)
- Escalating in volume. For new issues, we’ll compare the hourly event rate to the 95th percentile hourly event rate of other new issues in the project.
```

File: src/docs/product/alerts/create-alerts/high-priority-alerts.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+In summary, the high priority alert condition will notify you if an issue is:
+
+- Similar to issues that have been viewed & responded to quickly in the past, or likely to be actionable (read on for more details)
+- Escalating in volume: for new issues, we’ll compare the hourly event rate to the 95th percentile hourly event rate for new issues in the project
+
+**How are first event priority scores calculated?**
+
+We initially classify issues based on their log level. Low priority is assigned to issues with log levels 'debug' or 'info', while a 'fatal' log level is classified as high priority. Next, our algorithm employs a fine-tuned 'gte-small' text embeddings model to convert error messages into a vector of text embeddings. These embeddings, along with additional features like error handling status and the presence of a stacktrace, are then fed into a classification model. This model generates a final score that reflects the first-event severity and nature of the issue.
```
Comment (lizokm) [inline src/docs/product/alerts/create-alerts/high-priority-alerts.mdx:14]:
```suggestion
## How First Event Priority Scores Are Calculated

We initially classify issues based on their log level. Low priority is assigned to issues with log levels `debug` or `info`, while a `fatal` log level is classified as high-priority. Next, our algorithm employs a fine-tuned `gte-small` text embeddings model to convert error messages into a vector of text embeddings. These embeddings, along with additional features like error handling status and the presence of a stacktrace, are then fed into a classification model. This model generates a final score that reflects the first-event severity and nature of the issue.
```

File: src/docs/product/alerts/create-alerts/high-priority-alerts.mdx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+**Algorithm workflow in detail**
+
+1. Automatically classify issues with a log level of **debug** or **info** as low priority, and issues with a log level of **fatal** as high priority
+2. Use a fine-tuned version of the [gte-small](https://huggingface.co/thenlper/gte-small) text embeddings model to generate a vector of text embeddings from the error message
+3. Generate our full set of features and pass them to our classification model
+   1. Text embeddings
+   2. Whether or not the error was handled
+   3. Whether or not a stacktrace is present
+4. Return the score, which is then used to classify an issue as high or low priority
```
Comment (lizokm) [inline src/docs/product/alerts/create-alerts/high-priority-alerts.mdx:24]:
```suggestion
**Algorithm workflow in detail**

Here's a deeper dive into how the algorithm works:

1. Automatically classify issues with a log level of **debug** or **info** as low priority, and issues with a log level of **fatal** as high priority.
2. Use a fine-tuned version of the [gte-small](https://huggingface.co/thenlper/gte-small) text embeddings model to generate a vector of text embeddings from the error message.
3. Generate our full set of features and pass them to our classification model:
   - Text embeddings
   - Whether or not the error was handled
   - Whether or not a stacktrace is present
4. Return the score, which is then used to classify an issue as high or low priority.
```

**Review by lizokm — APPROVED** [review]
Made a few language suggestions, but otherwise this looks good!

File: src/docs/product/alerts/create-alerts/high-priority-alerts.mdx (hunk 1 above)
Comment (snigdhas) [reply src/docs/product/alerts/create-alerts/high-priority-alerts.mdx]: good call, updated this a bit!

**Comment by snigdhas** [comment]
Yup, we're still running the LA - thanks for asking!

**Comment by lizokm** [comment]
@snigdhas I'll delete this as soon as priority alerts go into GA :)

**Pull request closed without merging**